### PR TITLE
Update EIP-663: Update opcodes for SWAPN and DUPN

### PR DESCRIPTION
**Pull request by gumb0**
Update opcodes according to https://github.com/ethereum/execution-specs/blob/master/lists/evm/pending-opcodes.md.

### CODE REVIEW

**Review by chfast — APPROVED** [review]

**Comment by SamWilsn** [comment]
This'll require a review from @axic before it can be merged, or you can duplicate the EIP under a new number if you'd like to pursue these changes independently.

**Review by axic — APPROVED** [review]

**Pull request merged**
